src/tusb.c

```diff
@@ -136,6 +136,47 @@ void tusb_int_handler(uint8_t rhport, bool in_isr) {
   #endif
 }
 
+bool tusb_rhport_teardown(uint8_t rhport) {
+  bool ret = false;
+  // backward compatible call with tusb_init(void)
+  #if defined(TUD_OPT_RHPORT) || defined(TUH_OPT_RHPORT)
+    #if CFG_TUD_ENABLED && defined(TUD_OPT_RHPORT)
+    // deinit device stack, CFG_TUSB_RHPORTx_MODE must be defined
+    ret = ret || tud_deinit(TUD_OPT_RHPORT);
```

Comment:
Nitpick: typically we do
```suggestion
    ret |= tud_deinit(TUD_OPT_RHPORT);
```

Reply:
Replaced with assert